tests/config.nims

```diff
@@ -1,3 +1,4 @@
 switch("path", "$projectDir/../src")
 switch("define", "normDebug")
 switch("deepcopy", "on")
+switch("mm", "orc")
```

Comment:
@PhilippMDoerner interestingly enough, I couldn't make the tests run with the default memory manager. I would get a nil access error when closing the pool after extending it. No idea why that happened but I'm more or less OK to require ORC for Norm.

Reply:
Agreed on the basis that ORC is the default for nim 2.0 anyway